src/commands/topics.ts

```diff
@@ -515,20 +513,19 @@ LIMIT 10;`;
   });
 
   const editor = await vscode.window.showTextDocument(document, { preview: false });
-
-  // Try setting doc metadata so that the language server connects to the right region & user can submit right away
-  const loader = CCloudResourceLoader.getInstance();
-  const environments: CCloudEnvironment[] = await loader.getEnvironments();
-  if (!environments || environments.length === 0) {
-    logger.trace("No CCloud environments found, won't update metadata");
+  if (!isCCloud(cluster)) {
```

Comment:
Nit: Cleaner to 
```suggestion
  if (!isCCloud(topicViewProvider.kafkaCluster)) {
```

... and then to declare the `const cluster` / cast in the `else` block.

Or even better, don't use an else block (since if !ccloud() then we bailed already), then move the `const cluster` declaration to after the if / bail clause.

Reply:
🧼